<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>askBaseProb</class>
 <widget class="QDialog" name="askBaseProb">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Base Probabilities</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="3" column="1">
    <widget class="QPushButton" name="buttonSave">
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <spacer name="horizontalSpacer_7">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QComboBox" name="chooseMatrix">
       <item>
        <property name="text">
         <string>Standard Probability Matrix</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Normalized Probability Matrix</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Standard Logarithmic Matrix</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Normalized Logarithmic Matrix</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Save Matrix as:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <spacer name="horizontalSpacer_8">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="2" column="2">
      <widget class="QDoubleSpinBox" name="spinA">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="prefix">
        <string/>
       </property>
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.050000000000000</double>
       </property>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>C:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="6">
      <widget class="QDoubleSpinBox" name="spinC">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="prefix">
        <string/>
       </property>
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.050000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="6">
      <widget class="QDoubleSpinBox" name="spinT">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="prefix">
        <string/>
       </property>
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.050000000000000</double>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>A:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>G:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="7">
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="4">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>T:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="7">
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="2">
      <widget class="QDoubleSpinBox" name="spinG">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="prefix">
        <string/>
       </property>
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.050000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="3">
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="0" colspan="8">
      <widget class="QComboBox" name="chooseBase">
       <property name="toolTip">
        <string/>
       </property>
       <item>
        <property name="text">
         <string>Base Probability of 0.25</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>No base Probability</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Base Probability calculated from the input sequence</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Custom Base Probability</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="6">
      <widget class="QDoubleSpinBox" name="showTotal">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="keyboardTracking">
        <bool>true</bool>
       </property>
       <property name="maximum">
        <double>999.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="4" column="2" colspan="3">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Total:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
